src/FlowLinesLayer/FlowLinesLayer.ts

```diff
@@ -128,11 +129,11 @@ class FlowLinesLayer extends Layer<Props, LayerState, Context> {
 
   draw({ uniforms }: DrawParams) {
     const { gl } = this.context;
-    const { borderColor } = this.props;
+    const borderColor = this.props.borderColor || DEFAULT_BORDER_COLOR;
```

Comment:
Let's put the default value to `defaultProps`, same as with `borderThickness`.

Reply:
is  `|| DEFAULT_BORDER_COLOR` still necessary here?

Reply:
I guess it's because of TypeScript not understanding defaultProps

Reply:
Actually no, I had a quick look yesterday and none of the defaultProps were set at runtime. But need to investigate further.

Reply:
hm, that doesn't sound good

Reply:
I think it's related t the use a `CompositeLayer`. Possibly we should only set defaults on the top `FlowMapLayer`, not the sublayer props. `CompositeLayer.getSubLayerProps` does some magic